src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -977,13 +988,16 @@ protected override void ProcessRecord()
                 case relationshipSet:
                     string relationshipPath = null;
                     try
-                    {
+                    {   
+                        relationshipPath = ReplacePathWithLastLocationIfApplicable(relationshipPath);
+                        var initialPath = SessionState.Path.CurrentLocation.Path;
```

Comment:
You can revert the changes under `RELATIONSHIP_SUPPORTED` - the code is dead, and I've submitted a PR to delete it.

Reply:
Ok, will do.